core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -83,7 +83,7 @@
   /**
```

Comment:
As introduce page into blocklet, if can increase  BLOCKLET_SIZE_DEFAULT_VAL?  it would reduce I/O cost further.